openlibrary/catalog/merge/load_from_json.py

```diff
@@ -3,7 +3,7 @@
 
 import simplejson
 import re
-from normalize import normalize
+from openlibrary.catalog.onix.normalize import normalize
```

Comment:
This is a literal translation of what Python 2 would do with this relative import.  This is what is currently running in production because Python 2 looks for and findS normalize.py in the same directory as `__file__`.